include/imex/internal/PassUtils.h

```diff
@@ -18,11 +18,17 @@
 
 namespace imex {
 
+/// @return get ::mlir::IntegerAttr with given Value and bitwidth W
+template <int W = 64, typename T = int64_t>
```

Comment:
nit: `getIntegerAttr` takes `int64_t` anyway so this `typename T` useless

Reply:
Fixed.